.github/workflows/docker_image.yml

```diff
@@ -0,0 +1,70 @@
+---
+name: Docker Image
+
+# https://docs.github.com/en/actions/publishing-packages/publishing-docker-images
+# https://docs.docker.com/build/ci/github-actions/multi-platform
+# https://github.com/opencontainers/image-spec/blob/main/annotations.md
+
+on:
+  push:
+    branches: [ "master" ]
+    # Publish semver tags as releases.
+    tags: [ "*.*.*" ]
+  pull_request:
+    # Build docker image on pull requests to master. (but do not publish)
+    branches: [ "master" ]
```

Comment:
this duplicates what the existing workflow does: https://github.com/minetest/minetest/blob/master/.github/workflows/linux.yml#L155

We should either merge both or ensure this one only runs exactly when we need to publish a new image.

Reply:
What could be done is an additional workflow job in this workflow where the image is pulled and then run `--version` inside it.

Reply:
can't that be added to this workflow? the image should be present after the buildx action runs